src/pages/play-against-computer/container.js

```diff
@@ -28,9 +45,41 @@ export const mapDispatchToProps = (dispatch) => ({
     );
     dispatch(computerMove({ squareIndex, currentMovesPlayed }));
   },
+  jumpTo: (step) => dispatch(goToMove(step)),
+  scoreForPlayerX: (currentScore, winner) => {
+    if (winner === 'X') {
+      dispatch(setXScore(currentScore));
```

Comment:
Is this for getting the score or setting the score for player x?

Reply:
Same for the methods below

Reply:
For setting the score